robot/demos/RosAction/README_long.md

```diff
@@ -0,0 +1,50 @@
+1) create your catkin_ws
```

Comment:
delet this?

![delet_this_catto](https://user-images.githubusercontent.com/19865282/83930670-bd245880-a766-11ea-83e5-1c94dbb1cb03.png)

Reply:
(the whole file)

Reply:
done